lib/constable_web/views/announcement_view.ex

```diff
@@ -3,35 +3,33 @@ defmodule ConstableWeb.AnnouncementView do
 
   def json_interests(interests) do
     interests
-    |> Enum.map(&(%{name: &1.name}))
-    |> Poison.encode!
+    |> Enum.map(&%{name: &1.name})
+    |> Poison.encode!()
   end
 
   def comma_separated_interest_names(interests) when is_list(interests) do
     interests
-    |> Enum.map(&(&1.name))
+    |> Enum.map(& &1.name)
     |> Enum.join(",")
   end
 
   def comma_separated_interest_names(_), do: ""
 
-  def class_for("all", %{params: %{"all" => "true"}}) do
-    "selected"
-  end
-  def class_for("mine", %{params: %{"all" => "true"}}), do: nil
-  def class_for("mine", _) do
-    "mine selected"
```

Comment:
There was no `css` for `mine`, so I just left `selected` in the changes. Also added another `class_for` for when we are clicking on `"your announcements"`.